Bottom: 2c4f657a46d08c50030669e70e7bb93a0560c992
Top:    8a6d451beeab2490736eda03efa6ea4519da1f84
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:13:19 +0000

More work in progress


---

diff --git a/src/polygon1.c b/src/polygon1.c
index e4e3f4a..5be2e76 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2810,8 +2810,6 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
-static void clear_marks (POLYAREA * p);
-
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
